ST_grid.c

```diff
@@ -706,2357 +390,1206 @@ static void _init_grid_files(void)
+                       						"_allocate_accumulators(Gwf->prescribedfire");
+
+      				ForEachGroup(rg){
+        				gridCells[i][j]._Gwf->prescribedFire[rg] = (int *)
+          										   Mem_Calloc( SuperGlobals.runModelYears,
+                      										   sizeof(int) * SuperGlobals.runModelYears,
+                      										   "_allocate_accumulators(Gwf->prescribedFire)");
+      				}
+    			}
+  			}
+
+  			if (MortFlags.group) {
+    			gridCells[i][j]._Gestab = (struct stat_st *)
+             	  		  Mem_Calloc( Globals->grpCount,
+                         			sizeof(struct stat_st),
+                         			"_allocate_accumulators(Gestab)");
+
+				gridCells[i][j]._Gmort = (struct stat_st *)
+           		 		 Mem_Calloc( Globals->grpCount,
+                       				sizeof(struct stat_st),
+                      				"_allocate_accumulators(Gmort)");
+    			ForEachGroup(rg){
+      				gridCells[i][j]._Gestab[rg].s = (struct accumulators_st *)
+                     				Mem_Calloc( 1, sizeof(struct accumulators_st),
+                                				"_allocate_accumulators(Gestab[rg].s)");
+					gridCells[i][j]._Gmort[rg].s = (struct accumulators_st *)
+           							Mem_Calloc( GrpMaxAge(rg),
+                       							sizeof(struct accumulators_st),
+                      							"_allocate_accumulators(Gmort[rg].s)");
+				}
+  			}
+
+  			if (BmassFlags.sppb) {
+      			gridCells[i][j]._Spp = (struct stat_st *)
+               		   Mem_Calloc( Globals->sppCount,
+                           		   sizeof(struct stat_st),
+                          		   "_allocate_accumulators(Spp)");
+      			ForEachSpecies(sp){
+        			gridCells[i][j]._Spp[sp].s = (struct accumulators_st *)
+               			 		  Mem_Calloc( SuperGlobals.runModelYears,
+                           					sizeof(struct accumulators_st),
+                          					"_allocate_accumulators(Spp[sp].s)");
+				}
 
-/***********************************************************/
-static void _free_individuals(IndivType *head)
-{
-	//frees the memory allocated in the linked list of individuals pointed to by head
-	if (head->Next != NULL)
-		_free_individuals(head->Next); //recursively call itself in order to free the next obj in the list (Mem_Free() will end up getting called on the objects in the list from last to first)
-	Mem_Free(head);
+      			if (BmassFlags.indv) {
+        			gridCells[i][j]._Indv = (struct stat_st *)
+               		 		Mem_Calloc( Globals->sppCount,
+                           		 		sizeof(struct stat_st),
+                          		 		"_allocate_accumulators(Indv)");
+        			ForEachSpecies(sp){
+          				gridCells[i][j]._Indv[sp].s = (struct accumulators_st *)
+               				  		Mem_Calloc( SuperGlobals.runModelYears,
+                           						sizeof(struct accumulators_st),
+                          						"_allocate_accumulators(Indv[sp].s)");
+					}
+    			}
+  			}
```

Comment:
The bracket structure is correct, but irregularities in indentation make it look strange. Someone (possibly me) used 2 spaces followed by a tab before the bracket in question which is displayed differently by every document viewer. Again, this will be fixed by running clang formatter.